src/openads/domain/position/Position.js

```diff
@@ -120,4 +123,35 @@ export default class Position {
     }
     return this
   }
+
+  /**
+   * Update Position with given changes
+   * @param {{name: *, placement: *}} position
+   * @param {string} position.name
+   * @param {string} position.placement
+   * @param {string} position.segmentation
+   * @param {Array} position.sizes
+   * @returns {Position}
+   */
+  changeSegmentation ({ name = this._name, placement = this._placement, segmentation = this._segmentation, sizes = this._sizes } = {}) {
```

Comment:
**changeSegmentation** is the proper name for this method? Should not be something like **updatePosition** as far as the method changes not only _segmentation_ but also the rest of Position properties?

![](https://media.giphy.com/media/3o7buirYcmV5nSwIRW/giphy.gif)

Reply:
If you think in the possible properties to be changed for now...the global concept is Segmentation
Besides, updatePosition is less clear, update what?

Reply:
hahahaha, sure! I was trying to make myself clear. My concern with this method is to update several properties where one of them is called 'segmentation'... reading the code as it is could be confusing. IMHO